node/runtime/src/lib.rs

```diff
@@ -89,23 +89,64 @@ pub struct ApplyResult {
     pub authority_proposals: Vec<AuthorityProposal>,
 }
 
+#[allow(dead_code)]
+struct CallBackInfo {
```

Comment:
Here you should have num_results. This is the actual Callback. You also need to store account_id (of the receiver).

Reply:
Fixed in ca636d9